transform/transformTx.go

```diff
@@ -0,0 +1,49 @@
+package transform
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/stellar/starbridge/model"
+)
+
+// TODO this destination needs to be input from the input transaction on the remote chain from the memo or similar
+var destinationAccountForStellar = "GCBAA5476KARHPDSU6WFQTPXQOWX3QMXU4LF7JVZ2ZMWJ4OQEL7ZMV6G" // var destinationSecretKey = "SALR2RNJG55BBWTML2MKO5CXG5QDI4ZTSVDIA53XDWOU7QPOAEQNYUE2"
+
+// MapTxToChain converts a given transaction to the destination chain along with mapping the assets appropriately to the right contracts
+func MapTxToChain(tx *model.Transaction, destinationChain *model.Chain) (*model.Transaction, error) {
```

Comment:
this is the main transform function!

we convert from one chain to another. This is all driven by metadata. Since everything is defined for the chain and assets, this becomes very easy!